### ISSUE TITLE
[BUG] - blog post on PyData GPU extensibility has two issues

### ISSUE THREAD

**Issue by rgommers** [Quansight/Quansight-website#405]
### What site is this for?

Quansight Labs

### Expected behavior

I see two issues with https://labs.quansight.org/blog/pydata-extensibility-vision:
1. The header image is very weird and off-putting. I think it's actually better to remove that header image completely, because it takes up too much space and is highly unusual for a single blog post page. But if the design idea stays, a more suitable image should be chosen.
2. This was a two author post, Ivan Yashchuk (@ivanyashchuk) and me. Now I see Ivan has disappeared as a main author. Can we please fix that?

### Actual behavior

<img width="883" alt="image" src="https://user-images.githubusercontent.com/98330/182907105-d3f670c4-4ea4-4269-9e28-83c74bb33e56.png">

### How to Reproduce the problem?

Go to https://labs.quansight.org/blog/pydata-extensibility-vision

**Comment by bskinn** [comment]
@rgommers Your (2) is blocked pending #182. I'll add this as a TODO bullet in #364.

**Comment by bskinn** [comment]
Attn: @trallard @irinafumarel

**Comment by noatamir** [comment]
@rgommers just to clarify, would you like to discuss changing the design template so that all posts don't require a header (hero) image? Or it just bothers you in this case? 

Asking because if it's the latter we can replace it with a generic image using a template Irina already provided us with. 

But for the former it will require dev effort because at the moment, as best I recall, the post won't build without a hero image.

**Comment by rgommers** [comment]
I think it's either completely unnecessary or taking up too much vertical space in general. So it'd be good to improve that. But in this case it particularly jumps out, so if there's no appetite for design changes any time soon, replacing with the stock image would be nice.

**Comment by irinafumarel** [comment]
@rgommers I agree. I'll work with @noatamir to replace those placeholders with either proper illustrations or stock photos

**Comment by noatamir** [comment]
I agree as well. We'd be happy to get back to this ☺️. It's on our list of things we haven't finished tweaking and are waiting to get back to when time allows.

@MarsBarLee could you meanwhile open a PR to replace the hero with one of Irina's illustrations? I think this could be a good patch until long-term changes are implemented. (If I recall, we also discussed this when the PR was made, and you weren't sure about the cropped diagram either).

PS, yesterday I created a few more variations and placed them in [our shared folder](https://drive.google.com/drive/folders/10CFs2JW-mo9kCPpyMfPWAy6ef76SAe9Q?usp=sharing). So you have more options to work with 😉.

**Comment by rgommers** [comment]
Closing, as the main issue is tracked in gh-364 and the header image issue is no longer relevant.
